community/resources/docs-as-code.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: full-width
+group: community
+title: Docs as Code
+---
+
+To further expand documentation options for DevDocs, we offer a new opportunity for developers to contribute content through Community Engineering project repositories!
+
+Docs as Code is a new process that pulls content from public GitHub repositories and generates it as part of DevDocs. The DevDocs team helps you write add your content for the site. 
+
+{:.bs-callout .bs-callout-info}
+This is the first of many new options for expanding documentation for developers. We will add more information and examples coming soon.
+
+## Docs as Code benefits
```

Comment:
Don't use this term to describe the new concept we're introducing; "docs as code" already has a meaning in our industry.

Reply:
Renaming, see previous comment.